glib/src/signal.rs

```diff
@@ -13,6 +13,36 @@ use std::mem;
 use std::num::NonZeroU64;
 
 /// The id of a signal that is returned by `connect`.
+///
+/// ```no_run
+/// use glib::SignalHandlerId;
+/// use gtk::prelude::*;
+/// use std::cell::RefCell;
+///
+/// struct Button {
+///     widget: gtk::Button,
+///     connection: RefCell<Option<SignalHandlerId>>,
```

Comment:
```suggestion
///     clicked_handler_id: RefCell<Option<SignalHandlerId>>,
```

Or something like that